packages/panels/resources/views/components/page/index.blade.php

```diff
@@ -64,13 +64,17 @@
             ])
         >
             @if ($subNavigation)
-                {{ \Filament\Support\Facades\FilamentView::renderHook(\Filament\View\PanelsRenderHook::PAGE_SUB_NAVIGATION_SELECT_BEFORE, scopes: $this->getRenderHookScopes()) }}
+                <div class="md:hidden">
```

Comment:
@danharrin, this breaks the UI on mobile because it adds an additional item in the flex container which has a gap:
<img width="381" alt="Screenshot 2024-11-05 at 11 17 23" src="https://github.com/user-attachments/assets/6319f8c1-38bf-4104-8fab-c52fd585941c">

Reply:
Can the element be wrapped in a check to see if the render hook has output first then?

Reply:
That might be a good solution for cases where the render hook isn't used.
However, the issue would still be there when using the render hook and viewing the page on mobile I think.

Reply:
If the render hook is used the space is justified IMO

Reply:
Why would it be ok then?

Reply:
So there is a normal space between the content you put in the render hook and the select?

Reply:
Alright, let's apply that fix then.